modules/analysis-common/src/yamlRestTest/resources/rest-api-spec/test/search.query/11_match_field_match_only_text.yml

```diff
@@ -2,7 +2,7 @@
 
 "match query with stacked stems":
   - skip:
-      version: " - 2.99.99"
+      version: " - 2.11.99"
       reason: "match_only_text was added in 3.0"
```

Comment:
All versions <= skip_version will be skipped. If we add 2.12 then it would be skipped as well which we don't want since this change is backported to 2.x, which currently points 2.12.

Reply:
Yes, I was taking about the text "match_only_text was added in 2.12"